src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -85,7 +85,7 @@ public void handle(CellEditEvent<TransactionModel, String> t) {
 
                 updateTransactionTableView();
             } catch (StorageException e) {
-                System.out.println("Error editing transaction amount: " + e.getMessage());
+                setupErrorPopup("Error editing transaction amount.");
```

Comment:
Should probably keep the message. Latter definitely want to translate it but even the bare message should be useful.